The location <sys/fcntl.h> is wrong, it is just <fcntl.h>

--- a/src/lib/liblow.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/lib/liblow.c	2015-06-22 15:36:38.956046661 +0200
@@ -33,7 +33,7 @@
 #include <sys/types.h>     /* socket() */
 #include <sys/socket.h>    /* socket() */
 #include <sys/un.h>        /* struct sockaddr_un */
-#include <sys/fcntl.h>     /* O_RDONLY */
+#include <fcntl.h>         /* O_RDONLY */
 #include <sys/stat.h>      /* stat() */
 
 #ifdef  SIGTSTP         /* true if BSD system */
--- a/src/daemon/gpm.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/daemon/gpm.c	2015-06-22 15:44:46.825020280 +0200
@@ -29,7 +29,7 @@
 #include <signal.h>        /* SIGPIPE */
 #include <time.h>          /* time() */
 #include <sys/param.h>
-#include <sys/fcntl.h>     /* O_RDONLY */
+#include <fcntl.h>         /* O_RDONLY */
 #include <sys/wait.h>      /* wait()   */
 #include <sys/stat.h>      /* mkdir()  */
 #include <sys/time.h>      /* timeval */
